apps/api/src/app/bridge/e2e/health-check.e2e.ts

```diff
@@ -20,12 +21,13 @@ describe('Bridge Health Check #novu-v2', async () => {
         };
       });
     });
-    frameworkClient = new TestBridgeServer();
-    await frameworkClient.start({ workflows: [healthCheckWorkflow] });
+    const port = await getPort();
```

Comment:
This is another attempt to make it more robust in case the culprit is the Test Bridge Server. Since we have no more ideas on why it fails on CI, let's try a random port instead of the same one for each test.

Reply:
small suggestion:
i think it would be easier to create the port in the constructor instead of providing the port on every init.